src/inference/src/ie_core.cpp

```diff
@@ -182,13 +182,15 @@ void allowNotImplemented(F&& f) {
 
 ov::AnyMap flatten_sub_properties(const std::string& device, const ov::AnyMap& properties) {
     ov::AnyMap result = properties;
+    bool isVirtualDev = device.find("AUTO") != std::string::npos || device.find("MULTI") != std::string::npos ||
+                        device.find("HETERO") != std::string::npos;
     for (auto item = result.begin(); item != result.end();) {
         auto parsed = parseDeviceNameIntoConfig(item->first);
         if (!item->second.is<ov::AnyMap>()) {
             item++;
             continue;
         }
-        if (device.find(parsed._deviceName) != std::string::npos) {
+        if (device == parsed._deviceName) {
```

Comment:
does this cover GPU.* case?

Reply:
Do you mean the device properties in the config like {"GPU.1", "NUM_STREAMS 4"} could be passed to GPU plugin correctly?

Reply:
yes, I mean be passed as the secondary property to AUTO/MULTI, maybe we can update relevant document to list all the situations and expected results, like for me with 2 GPUs,  I want to know if -d MULTI:CPU, GPU -nstreams CPU:3,GPU.0:3,GPU.1:4  is a valid setting from benchmark, and if all the settings can take effect in each hw plugins